src/main/resources/reference.conf

```diff
@@ -48,67 +56,84 @@ common {
 }
 
 interactions {
+  scorethreshold = 0
+  target-etl {
+    format = ${common.output-format}
+    path = ${common.output}"/targets"
+  }
   rnacentral {
-    format = "json"
-    path = "gs://ot-snapshots/etl/inputs/20.11/jsonl/interactions/rna-central/"
+    format = "csv"
+    path = "gs://ot-snapshots/etl/inputs/21.02/jsonl/interactions-new/rna_central_ensembl.tsv"
+    options = [
+      {k: "sep", v: "\\t"},
+      {k: "header", v: "true"}
+    ]
   }
   humanmapping {
-    format = "json"
-    path = "gs://ot-snapshots/etl/inputs/20.11/jsonl/interactions/human-mapping/"
+    format = "csv"
+    path = "gs://ot-snapshots/etl/inputs/21.02/jsonl/interactions-new/HUMAN_9606_idmapping.dat.gz"
+    options = [
+      {k: "sep", v: "\\t"},
+      {k: "header", v: "true"}
+    ]
   }
   ensproteins {
-    format = "json"
-    path = "gs://ot-snapshots/etl/inputs/20.11/jsonl/interactions/ensembl-protein-2020-11-26.json.gz"
+    format = "csv"
+    path = "gs://ot-snapshots/etl/inputs/21.02/jsonl/interactions-new/Homo_sapiens.GRCh38.chr.gtf.gz"
+    options = [
+      {k: "sep", v: "\\t"},
+      {k: "header", v: "true"}
```

Comment:
We should add another k,v here: `{k: "comment", v: "#"}` to have Spark remove the comment lines at the top of the file automatically. 

From the docs
```
   * <li>`comment` (default empty string): sets a single character used for skipping lines
   * beginning with this character. By default, it is disabled.</li>
   * ```
We can then remove the manual filtering in Interactions